csfieldguide/utils/custom_converter_templates/chapter-section/heading.html

```diff
@@ -1,22 +1,20 @@
-<{{ heading_type }} id="{{ title_slug }}">
-<span class="section_number">
+<div class="section-heading">
+{%- if heading_level < 4 -%}
+<div class="section-heading-number">
```

Comment:
>     * Headings with numbers longer than 3 levels now don't show a number.

I don't think this is taking into account the chapter number itself, so if `Headings with numbers longer than 3 levels now don't show a number` it would only be # and ## headings in markdown that get numbers. Hope that makes some sense, point is; *headings numbers as long as 4 levels of numbers still show a number*

![image](https://user-images.githubusercontent.com/25916475/59570286-0ad52a80-90ea-11e9-8310-b48e2b9094ee.png)

Reply:
Nice catch, will fix it later today.

Reply:
You seem very busy @JackMorganNZ so I fixed it. Should be good now